src/features/SearchResults/index.js

```diff
@@ -0,0 +1,116 @@
+import { useLocation } from "react-router-dom/cjs/react-router-dom";
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { TilesContainer } from "../../common/Tiles/MovieTilesContainer/styled";
+import { TilesHeader } from "../../common/Tiles/TilesHeader/styled";
+import noMovieImage from "../../images/no-movie-image.svg";
+import noPersonImage from "../../images/not-found-person.svg";
+import Pagination from "../../common/Pagination";
+import LoadingPade from "../../common/LoadingPage";
+import { MovieTile } from "../../common/Tiles/MovieTilesContainer/MovieTile";
+import PersonsTile from "../../common/Tiles/PersonTile/PersonsTile";
+import NoResults from "../../common/NoResults";
+import Error from "../../common/ErrorPage";
+import { Container } from "../MainPage/Container";
+import { useSearchResults } from "./useSearchResults";
+import { Tiles } from "../../common/Tiles/PersonTile/TilesContainer";
+import { useGenres } from "./useGenres";
+import { useQueryParameter } from "../../common/NavigationBar/SearchBar/queryParameters";
+import { selectCurrentPage } from "../../common/Pagination/paginationSlice";
+import {
+  fetchCurrentPage,
+  selectSearchMoviesData,
```

Comment:
I think selectSearchMoviesData and selectSearchMoviesStatus are not used in this file, so you could delete them.